docker-assets/Gemfile

```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+gem "amazon_ssa_support", ">0", :require => "amazon_ssa_support", :git => "https://github.com/ManageIQ/amazon_ssa_support.git", :branch => "master"
+gem "manageiq-gems-pending", ">0", :require => "manageiq-gems-pending", :git => "https://github.com/ManageIQ/manageiq-gems-pending.git", :branch => "master"
+gem "manageiq-smartstate", "0.1.5", :require => "manageiq-smartstate"
```

Comment:
@bdunne Do we need to update the version of manageiq-smartstate gem to the latest 0.2.1?